internal/experiment/webconnectivityqa/badssl.go

```diff
@@ -89,6 +89,10 @@ func badSSLWithUnknownAuthorityWithInconsistentDNS() *TestCase {
 		},
 		ExpectErr: false,
 		ExpectTestKeys: &testKeys{
+			BodyLengthMatch:       true,
+			StatusCodeMatch:       true,
+			HeadersMatch:          true,
+			TitleMatch:            true,
```

Comment:
This is not a regression. Web Connectivity LTE includes code to periodically and opportunistically perform DNS-over-HTTPS (DoH) measurements. The first run in a list of URLs always uses DoH. After the first DoH attempt, we do not try again for a given number of seconds. Because the QA suite always runs with the first run, we are getting and using a valid IP address for the domain and hence we're able to fetch the right webpage.

Note that, even without DoH, if we can communicate with the test helper, we're getting legit IP address that we are going to use. Therefore, this test case would _also_ fetch the correct web page w/o DoH.